TarSCM/scm/git.py

```diff
@@ -251,7 +251,7 @@ def prepare_working_copy(self):
         # between multiple services
         org_clone_dir  = self.clone_dir
         self.clone_dir = self.repodir
-        command = ['git', 'clone', self.repocachedir, self.url, self.clone_dir]
+        command = ['git', 'clone', '--no-checkout']
```

Comment:
So I did:

```
osc co home:bluca:branches:network:messaging:zeromq:git-draft/zproject
cd home\:bluca\:branches\:network\:messaging\:zeromq\:git-draft/zproject
osc service localrun
osc build
echo lol >> zproject/zproject_android.gsl
osc build
```

And the "lol" is there in the built package at the bottom of the file.
The _service uses tar_scm: https://build.opensuse.org/package/show/home:bluca:branches:network:messaging:zeromq:git-draft/zproject

Did I miss something?